llama_stack/providers/remote/inference/vllm/vllm.py

```diff
@@ -229,7 +230,11 @@ def __init__(self, config: VLLMInferenceAdapterConfig) -> None:
 
     async def initialize(self) -> None:
         log.info(f"Initializing VLLM client with base_url={self.config.url}")
-        self.client = AsyncOpenAI(base_url=self.config.url, api_key=self.config.api_token)
+        self.client = AsyncOpenAI(
+            base_url=self.config.url,
+            api_key=self.config.api_token,
+            http_client=None if self.config.tls_verify else httpx.AsyncClient(verify=False),
```

Comment:
This should be `verify=True` if `tls_verify` is true?

Reply:
True is already the default.